develop/components/pages/BreadcrumbPage.tsx

```diff
@@ -1,17 +1,36 @@
-import * as React from "react";
-import { Breadcrumb } from "../../../src/Breadcrumb/Breadcrumb";
+import React from "react";
+import H from "history";
+import { useHistory } from "react-router";
+import { Breadcrumb, BreadcrumbItem } from "../../../src/Breadcrumb/Breadcrumb";
 import Highlight from "react-highlight";
 const docMD: string = require("../../../src/Breadcrumb/readme.md");
 
-const userIcon: JSX.Element = (
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 170 170">
-        <title>regular_black</title>
-        <path d="M149.1,165h-6V132.3c0-18.8-14.2-34.8-32.5-36.8l-24,15.9a3,3,0,0,1-3.3,0L59.2,95.5C41,97.7,26.9,113.6,26.9,132.3V165h-6V132.3c0-22.5,16.7-40.9,38.8-42.9a3,3,0,0,1,1.9.5L85,105.3l23.2-15.4a3,3,0,0,1,1.9-.5c21.9,1.8,39,20.7,39,42.9Z" />
-        <path d="M85,86.4A31.7,31.7,0,0,1,53.4,54.8V36.6a31.6,31.6,0,1,1,63.3,0V54.8A31.7,31.7,0,0,1,85,86.4Zm0-75.5A25.7,25.7,0,0,0,59.4,36.6V54.8a25.6,25.6,0,0,0,51.3,0V36.6A25.7,25.7,0,0,0,85,10.9Z" />
+const houseIcon: JSX.Element = (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+            d="M1092,444.005a0.978,0.978,0,0,1-.29.709,0.841,0.841,0,0,1-.19.132A2.194,2.194,0,0,1,1090,446h-1v8a2,2,0,0,1-2,2h-14a2,2,0,0,1-2-2v-8h-1a2,2,0,0,1-1.99-1.942c0-.015-0.01-0.03-0.01-0.045V444h0a0.97,0.97,0,0,1,.29-0.666l10.9-10.906a0.53,0.53,0,0,1,.1-0.143,1.037,1.037,0,0,1,1.42,0,0.535,0.535,0,0,1,.1.144l10.9,10.905a0.97,0.97,0,0,1,.29.666h0v0Zm-13,10h2v-5h-2v5Zm1-19.619L1070.38,444H1073v10h4v-6a1,1,0,0,1,1-1h4a1,1,0,0,1,1,1v6h4V444h2.62Z"
+            transform="translate(-1068 -432)"
+        />
+    </svg>
+);
+const musicIcon: JSX.Element = (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+            d="M1088,312a4,4,0,1,1,2-7.445v-14.31l-14,4.49V307a0.925,0.925,0,0,1-.07.332,3.691,3.691,0,0,1,.07.668,4.046,4.046,0,1,1-2-3.445V294a0.9,0.9,0,0,1,1.14-.971l15.62-5.009a0.959,0.959,0,0,1,.2-0.011c0.01,0,.03-0.009.04-0.009a0.146,0.146,0,0,1,.02,0,0.941,0.941,0,0,1,.89.6,0.015,0.015,0,0,1,.01.016c0.01,0.025.03,0.044,0.04,0.07,0.01,0.054,0,.106.01,0.16a0.792,0.792,0,0,1,.03.154v19A4,4,0,0,1,1088,312Zm0-6a2,2,0,1,0,2,2A2,2,0,0,0,1088,306Zm-16,0a2,2,0,1,0,2,2A2,2,0,0,0,1072,306Z"
+            transform="translate(-1068 -288)"
+        />
     </svg>
 );
 
 const BreadcrumbPage: React.FunctionComponent = () => {
```

Comment:
```suggestion
const BreadcrumbPage: React.FunctionComponent = (): React.ReactElement<void> => {
```

Reply:
I don't think it's needed here since we defined as `FunctionComponent` which will prevent you from returning anything other than a `ReactElement`. It's similar to this:
```typescript
const Comp: React.FC = (): boolean => {
    return true;
}
```
Typescript will throw the following error:
```terminal
Error: Type '() => boolean' is not assignable to type 'FC<{}>'.
```

Reply:
I'm only trying to avoid using it because once we start adding these unimportant additions it will screw up the alignment

Reply:
While it simply adds the return type if the function which is necessary, I noticed we don't have it in either of the demo pages and hence i will agree with you on the structure.